geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/RedisRestoreKeyExistsException.java

```diff
@@ -17,7 +17,7 @@
 /**
  * An exception thrown when the key being restored already exists.
  */
-public class RedisRestoreKeyExistsException extends RuntimeException {
+public class RedisRestoreKeyExistsException extends RedisException {
```

Comment:
Very minor point, but might this class be better located in the `org.apache.geode.redis.internal.data` package?

Reply:
Done